android/src/main/java/io/mosip/tuvali/transfer/Util.kt

```diff
@@ -9,8 +9,14 @@ import java.security.MessageDigest
 import java.util.zip.GZIPInputStream
 import java.util.zip.GZIPOutputStream
 import io.mosip.tuvali.openid4vpble.Openid4vpBleModule
+import io.mosip.tuvali.transfer.ArraySize.FourBytes
+import io.mosip.tuvali.transfer.ArraySize.TwoBytes
+import java.nio.ByteBuffer
 
-
+enum class ArraySize {
```

Comment:
Should we call this `ByteCount` instead of `ArraySize`? To me arraysize is not giving correlation of what this enum is about.